app/scripts/views/reset_password.js

```diff
@@ -8,6 +8,7 @@ define(function (require, exports, module) {
   const AuthErrors = require('lib/auth-errors');
   const BaseView = require('views/base');
   const Cocktail = require('cocktail');
+  const FlowBeginMixin = require('views/mixins/flow-begin-mixin');
```

Comment:
Actually, that *should* work out okay automagically because the resume token includes the flow id and the flow begin time. I think.